pages/_language/events/_event/index.vue

```diff
@@ -1,41 +1,100 @@
 <template>
-  <main class="event">
-    <h2 class="event__title">{{ event.title }}</h2>
-    <nav>
-      <ul>
-        <li v-for="{ locale } in event._allTitleLocales" :key="locale">
-          <nuxt-link
-            rel="alternate"
-            :hreflang="locale"
-            :lang="locale"
-            :to="`/${locale}/events/${params.event}`"
-          >
-            {{ locale }}
-          </nuxt-link>
-        </li>
-      </ul>
-    </nav>
-
-    <ul class="event__relevant-chapters">
-      <li
-        class="chapter-preview"
-        v-for="chapter in event.relevantChapters"
-        :key="chapter.slug"
+  <div class="event">
+    <header class="event-header">
+      <div class="event-header__intro">
+        <h1 class="event__name">
+          <span class="event__location">{{ event.location }}</span>
+          {{ event.name }}
+        </h1>
+        <nav class="language-switch">
+          <ul class="language-switch__list">
+            <li
+              v-for="{ locale } in event._allNameLocales"
+              :key="locale"
+            >
+              <nuxt-link
+                class="language-switch__link"
+                :class="{ 'language-switch__link--active': params.language === locale }"
+                rel="alternate"
+                :hreflang="locale"
+                :lang="locale"
+                :to="`/${locale}/events/${params.event}`"
+              >
+                {{ locale }}
+              </nuxt-link>
+            </li>
+          </ul>
+        </nav>
+      </div>
+      <div class="event-header__body">
+        <h2 class="event-header__copy">
+          <span class="event-header__date">
+            <time :datetime="event.startDate">{{ new Date(event.startDate).getDate() }}</time>  -
+            <time :datetime="event.endDate">{{ new Date(event.endDate).getDate() }}</time>
+            July 2020 ({{ event.timezone }})
+          </span>
+          <span class="event-header__title">
+            {{ event.location }} {{ event.name }}
+          </span>
+        </h2>
+      </div>
+    </header>
+    <main class="event__body">
+      <aside class="wwa-mention">
+        <img class="wwa-mention__image" src="/favicon-96x96.png" width="32" height="32">
```

Comment:
This element is missing an `alt`-tag.

Reply:
Good catch, since it's just visual I added an empty `alt`!